include/itkMaxPhaseCorrelationOptimizer.h

```diff
@@ -88,6 +88,12 @@ class ITK_TEMPLATE_EXPORT MaxPhaseCorrelationOptimizer
   };
   itkGetConstMacro( PeakInterpolationMethod, PeakInterpolationMethod );
   void SetPeakInterpolationMethod( const PeakInterpolationMethod peakInterpolationMethod );
+  friend std::ostream&
+  operator<<( std::ostream& os, const PeakInterpolationMethod& pim )
+  {
+    os << static_cast< typename std::underlying_type< PeakInterpolationMethod >::type >( pim );
+    return os;
+  }
```

Comment:
Interesting!

Reply:
I got it from some StackOverflow discussion.